staging/src/k8s.io/apiserver/pkg/endpoints/openapi/openapi.go

```diff
@@ -139,7 +139,11 @@ func friendlyName(name string) string {
 }
 
 func typeName(t reflect.Type) string {
-	return fmt.Sprintf("%s.%s", t.PkgPath(), t.Name())
+	path := t.PkgPath()
+	if strings.Contains(path, "/vendor/") {
+		path = path[strings.Index(path, "/vendor/")+len("/vendor/"):]
```

Comment:
This makes the `typeName` the same as the other [helper](https://github.com/kubernetes/kubernetes/blob/0cff83931739cb7dbf344b502540d7e342eeaf04/staging/src/k8s.io/apiserver/pkg/server/openapi/openapi.go#L118-L121) used in another package.

I can leave a todo to move the util to a common place. cc @mbohlool